recipes/cmyt/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "cmyt" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cmyt-{{ version }}.tar.gz
+  sha256: 9090d477cb9e11c1067887489112ffc6fb5fbaaa994dd111167c04d90dd51e34
+
+build:
+  number: 0
+  skip: true   # [py<36]
```

This looks like it's pure Python and could be noarch? If so, add this here and adjust the dependence on `typing-extensions` below:
```suggestion
  noarch: python
```